crates/uv-python/fetch-download-metadata.py

```diff
@@ -255,8 +255,14 @@ async def _fetch_downloads(self, pages: int = 100) -> list[PythonDownload]:
                 # Sort the assets to ensure deterministic results
                 row["assets"].sort(key=lambda asset: asset["browser_download_url"])
                 for asset in row["assets"]:
+                    # On more recent versions we don't have files with hashes anymore,
+                    # but on older versions GitHub didn't backfill the digest.
+                    if digest := asset["digest"]:
```

Comment:
I think we only dropped the individual `.sha256` files, but the combined `SHA256SUMS` file is still around, and that’s the one the `fetch-python-metadata.py` script uses.